iglu-registry/0.3.0/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base:0.1.0
+LABEL maintainer="Snowplow Analytics Ltd. <support@snowplowanalytics.com>"
+
+# The version of the registry to download
+ENV IGLU_REGISTRY_VERSION="0.3.0"
+
+# The name of the archive to download
+ENV ARCHIVE="iglu_server_${IGLU_REGISTRY_VERSION}.zip"
+
+# Install Iglu Registry
+RUN mkdir -p /tmp/build && \
+    cd /tmp/build && \
+    wget -q http://dl.bintray.com/snowplow/snowplow-generic/${ARCHIVE} && \
+    unzip -d ${SNOWPLOW_BIN_PATH} ${ARCHIVE} && \
+    cd /tmp && \
+    rm -rf /tmp/build && \
+    mv ${SNOWPLOW_BIN_PATH}/iglu-server-* ${SNOWPLOW_BIN_PATH}/iglu_server.jar
+
+# Install postgresql
+RUN apk update && \
+    apk add postgresql-client
+
+COPY ./scripts/super_api_key_init.sh ${SNOWPLOW_BIN_PATH}
+RUN chmod +x ${SNOWPLOW_BIN_PATH}/super_api_key_init.sh && \
+	sed -i -e 's/\r$//' ${SNOWPLOW_BIN_PATH}/super_api_key_init.sh
+
+WORKDIR $SNOWPLOW_BIN_PATH
+
+EXPOSE 8080
```

Comment:
I'd rather specify the entrypoint here than in the docker compose example

Reply:
Entrypoint includes `-Dconfig.file=<config_file>` which contains db configuration. Moving entrypoint here requires there is a postgres instance with that db credentials. Otherwise iglu registry will throw MissingConfiguration exception and stop. In short, running iglu-registry without postgres is useless. Should I still include entrypoint here @BenFradet ?

Reply:
yes, you need an entrypoint so you can control how the app is launched

and no you don't need to specify the config file in the entrypoint

as I said before, cf other docker images

Reply:
Ah, right, I was always overriding default config.

Reply:
Hey @chuwy there are two concerns we need to decide on about dockerizing iglu-registry. They came out after moving `ENTRYPOINT` from docker-compose to Dockerfile, per @BenFradet 's feedback.

1) Would that be possible to add another issue to r9 for adding `--config` CLI parameter to iglu-registry? Currently only way to provide custom config is through system property, `-Dconfig.file=<path_to_config_file>` and this config approach disables us to make real use of `CMD`  instruction when writing Dockerfile. Other dockerized apps have a `--config` CLI parameter and people can provide custom config file by mounting a config directory and providing this `--config` option.

2) If we are positive for first question, can we also add `--help` CLI parameter to iglu-registry?

Reply:
I like this approach

Reply:
@oguzhanunlu sure, it makes sense!